t/lib/OpenQA/SeleniumTest.pm

```diff
@@ -173,7 +173,7 @@ sub wait_for_ajax {
         sleep $check_interval;
         $slept = 1;
     }
-    pass("Wait for jQuery successful$msg");
+    note "Wait for jQuery successful$msg";
```

Comment:
I assume this won't show in CircleCI and potentially contribute to a timeout with no output - the comment doesn't mention it, and I think that's potentially important to be aware of

Reply:
sorry. I don't know what you mean. However, the logs in circleCI certainly still show a message.

Reply:
Unlike `diag` a `note` is not included in junit output, see https://perldoc.perl.org/Test::More#note

Reply:
sure, that's the junit output which is only about real tests executed

Reply:
oh, now I understand what you mean. If there is simply no other test line output for a long enough time then circleCI might abort the job as there was no output received even though there is debugging log that is forwarded into another target. Yes, this might fail tests in those cases. However then something else is broken because no way should there be individual steps that take longer than 10 minutes, the default "no output timeout" in circleCI